packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.tsx

```diff
@@ -325,88 +299,131 @@ export default function AccessRequirementList(
+          <DialogContent className={'AccessRequirementList login-modal '}>
             <StandaloneLoginForm
               sessionCallback={() => {
                 window.location.reload()
               }}
             />
-          </Box>
-        )
-        break
-      case RequestDataStep.COMPLETE:
-        renderContent = <RequestDataAccessSuccess onHide={onHide} />
-        break
-      case RequestDataStep.SHOW_ALL_ARS:
-      default:
-        renderContent = content
-        dialogActions = (
-          <Button variant="contained" onClick={onHide}>
-            Close
-          </Button>
-        )
-    }
+          </DialogContent>
+        </>
+      )
+      break
+    case RequestDataStep.COMPLETE:
+      renderContent = <RequestDataAccessSuccess onHide={onHide} />
+      break
+    case RequestDataStep.SHOW_ALL_ARS:
+    default:
+      renderContent = (
+        <>
+          <DialogBaseTitle title={dialogTitle} onCancel={onHide} />
+          <DialogContent>
+            <DialogSubsectionHeader sx={{ mt: 0 }}>
+              What is this request for?
+            </DialogSubsectionHeader>
+            <Typography variant={'body1'} component={'span'}>
+              {requestDetails}
+            </Typography>
+            <DialogSubsectionHeader>
+              What do I need to do?
+            </DialogSubsectionHeader>
+            <AuthenticatedRequirement />
+            {anyARsRequireCertification && <CertificationRequirement />}
+            {anyARsRequireProfileValidation && <ValidationRequirement />}
+            {anyARsRequireTwoFactorAuth && <TwoFactorAuthEnabledRequirement />}
+            {sortedAccessRequirementIds
+              ?.map(id => accessRequirements!.find(ar => id === String(ar.id))!)
+              ?.map(accessRequirement => {
+                return (
+                  <AccessRequirementListItem
+                    key={accessRequirement.id}
+                    accessRequirement={accessRequirement}
+                    subjectId={subjectId}
+                    subjectType={subjectType}
+                    onHide={onHide}
+                    onRequestAccess={accessRequirement => {
+                      const nextStep = isSignedIn
+                        ? RequestDataStep.UPDATE_RESEARCH_PROJECT
+                        : RequestDataStep.PROMPT_LOGIN
+                      requestDataStepCallback({
+                        managedACTAccessRequirement: accessRequirement,
+                        step: nextStep,
+                      })
+                    }}
+                  />
+                )
+              })}
+          </DialogContent>
+          <DialogActions>
+            {customDialogActions ? (
+              customDialogActions
+            ) : (
+              <Button variant="contained" onClick={onHide}>
+                Close
+              </Button>
+            )}
+          </DialogActions>
+        </>
+      )
+  }
+
+  if (renderAsModal) {
```

Comment:
Now decide if we want to `renderAsModal` (though I think we should probably just get rid of this prop, since the `false` case doesn't really work)